.github/workflows/testing-linux.yml

```diff
@@ -1,4 +1,4 @@
-name: CI Pipeline Tests Linux
+name: Tests Linux
```

Comment:
Suggestion to name them "Unit Tests Linux" to make it clear this will not run any integration or system tests.

Reply:
Ah fair enough